sw/host/opentitantool/src/main.rs

```diff
@@ -112,6 +119,14 @@ fn main() -> Result<()> {
 
     let transport = backend::create(&opts.backend_opts)?;
 
+    if let Some(arg) = &opts.bootstrap {
+        let mut args = vec![OsString::from("bootstrap"); 1];
+        args.extend(shellwords::split(arg)?.iter().map(OsString::from));
+        let bootstrap = command::bootstrap::BootstrapCommand::from_iter(&args);
```

Comment:
If you wanted to be really cool one way you could achieve this is
```rust
BootstrapCommand::from_iter(iter::once("bootstrap".to_string()).chain(shellwords::split(arg)?).map(OsString::from))
```
but that might be too wordy.